src/internal/packager/helm/chart.go

```diff
@@ -119,19 +87,50 @@ func (h *Helm) InstallOrUpgradeChart() (types.ConnectStrings, string, error) {
 			output, err = h.upgradeChart(lastRelease, postRender)
 		} else {
 			// 😭 things aren't working
-			return nil, "", fmt.Errorf("unable to verify the chart installation status: %w", histErr)
+			return fmt.Errorf("unable to verify the chart installation status: %w", histErr), true
```

Comment:
@Racer159 this was what caused the change to `Retry` needing to have a `fatal` check. I don't love it but it is possibly useful for other things?

Reply:
We could not fatal at this point as well - this could indicate something that will not recover like having a bad kube cotext but it could also be a temporary issue if the network hiccups during the release history query